--- config.h	Tue Jul  9 10:03:30 2013
+++ config.h	Tue Jul  9 09:58:13 2013
@@ -5,19 +5,19 @@
 
 /* Define to 1 if you have the declaration of `acosh', and to 0 if you don't.
    */
-#define HAVE_DECL_ACOSH 0
+#define HAVE_DECL_ACOSH 1
 
 /* Define to 1 if you have the declaration of `asinh', and to 0 if you don't.
    */
-#define HAVE_DECL_ASINH 0
+#define HAVE_DECL_ASINH 1
 
 /* Define to 1 if you have the declaration of `atanh', and to 0 if you don't.
    */
-#define HAVE_DECL_ATANH 0
+#define HAVE_DECL_ATANH 1
 
 /* Define to 1 if you have the declaration of `expm1', and to 0 if you don't.
    */
-#define HAVE_DECL_EXPM1 0
+#define HAVE_DECL_EXPM1 1
 
 /* Define to 1 if you have the declaration of `feenableexcept', and to 0 if
    you don't. */
@@ -57,7 +57,7 @@
 
 /* Define to 1 if you have the declaration of `log1p', and to 0 if you don't.
    */
-#define HAVE_DECL_LOG1P 0
+#define HAVE_DECL_LOG1P 1
 
 /* Define to 1 if you have the <dlfcn.h> header file. */
 #undef HAVE_DLFCN_H
