--- ./LibCore/Assert/Wm5Assert.h
+++ ./LibCore/Assert/Wm5Assert.h
@@ -11,7 +11,7 @@
 
 #include "Wm5CoreLIB.h"
 
-#ifdef WM5_USE_ASSERT
+#if 0
 //----------------------------------------------------------------------------
 // Use WM5 asserts with file/line tracking.
 //----------------------------------------------------------------------------
@@ -47,7 +47,7 @@
 //----------------------------------------------------------------------------
 // Use standard asserts.
 //----------------------------------------------------------------------------
-#define assertion(condition, format, ...) assert(condition)
+#define assertion(condition, ...) assert(condition)
 //----------------------------------------------------------------------------
 #endif
 
